move-mutator/src/operators/binary.rs

```diff
@@ -85,8 +89,33 @@ impl MutationOperator for Binary {
             },
         };
 
+        let is_left_exp_zero = contains_value_zero(self.exps[0].exp.as_ref());
+        let is_right_exp_zero = contains_value_zero(self.exps[1].exp.as_ref());
+
         ops.into_iter()
             .filter(|v| cur_op != *v)
+            .filter(|v| match cur_op {
+                // All below mutants would lead to the same code logic and would become
```

Comment:
Could we add a comment that all of this is true only for unsigned integers, and once we have signed integers, we may want this to be type-based?